packages/adena-extension/src/components/transfer/address-input/address-input.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useRef } from 'react';
+import { AddressInputWrapper } from './address-input.styles';
+import TransferAddressBookIcon from '@assets/transfer-address-book.svg';
+import CancelIcon from '@assets/cancel-small.svg';
+import AddressBookList from '@components/transfer/address-book-list/address-book-list';
+
+export interface AddressInputProps {
+  opened: boolean;
+  hasError: boolean;
+  selected: boolean;
+  selectedName: string;
+  selectedDescription: string;
+  address: string;
+  errorMessage?: string;
+  addressBookInfos: {
+    addressBookId: string;
+    name: string;
+    description: string;
+  }[];
+  onClickInputIcon: (selected: boolean) => void;
+  onChangeAddress: (address: string) => void;
+  onClickAddressBook: (addressBookId: string) => void;
+}
+
+const AddressInput: React.FC<AddressInputProps> = ({
```

Comment:
<img width="1181" alt="Screenshot 2023-04-21 at 7 32 59 PM" src="https://user-images.githubusercontent.com/50394458/233615049-dfb5b913-49af-4a4d-8950-ae48fe23e2b1.png">
I think the borders are weird